DesignSafe-CI/DS-User-Guide
# feat: edit on GitHub

## Overview

Show the "Edit on GitHub" link on each page.

## Status

- [x] should say "Suggest an update via GitHub"
    <sup>use [TACC/TACC-Docs#v0.15.0](https://github.com/TACC/TACC-Docs/releases/tag/v0.15.0)</sup>
- <del>consider improving docs like TACC-Docs did</del>
    - <sub>https://github.com/TACC/TACC-Docs/pull/70</sub>
    - <sub>https://github.com/TACC/TACC-Docs/pull/77</sub>
    
    <ins>done via #123</ins>

## Related

- requires https://github.com/TACC/TACC-Docs/pull/57
- fixed (in Docker) by #124

## Changes

- **added** config values
- ~~**changed** TACC-Docs version~~

## Testing

0. Be logged into GitHub.
1. Open https://localhost:8000/.
2. See "Edit on GitHub" link at top right.
3. Click link.
4. Verify GitHub edit page for that file appears.

## UI

| before | after |
| - | - |
| <img width="960" alt="before" src="https://github.com/user-attachments/assets/3993584f-5e51-431f-a2da-7024b7f809f6" /> | <img width="960" alt="after" src="https://github.com/user-attachments/assets/4085d011-23fc-42c9-be0d-e8db74d1577b" /> |

https://github.com/user-attachments/assets/614dfbe1-9807-44e5-816d-b8fdf8b2136d

https://github.com/user-attachments/assets/b643c20c-7c64-43e0-9cdb-3ac342612c15

## Notes

DS User Guide was made public a while back, but adding "Edit on GitHub" link was delayed until the repo had good enough edit and test instructions.
